interface/VideoEncoderDefs.h

```diff
@@ -407,12 +406,6 @@ typedef struct VideoConfigBitRate {
     VideoRateControlParams rcParams;
 }VideoConfigBitRate;
 
-typedef struct VideoConfigAVCIntraPeriod {
```

Comment:
This is a interface change. We should increase micro api version number https://github.com/01org/libyami/blob/master/configure.ac#L9